Lib/asyncio/selector_events.py

```diff
@@ -246,8 +246,16 @@ def _accept_connection2(self, protocol_factory, conn, extra,
                 self.call_exception_handler(context)
 
     def _ensure_fd_no_transport(self, fd):
+        fileno = fd
```

Comment:
Please not make a new `fileno` local variable but reuse `fd` in the rest of function -- like `selectors._fileobj_to_fd` does.

Reply:
`fd` is used later in this function to format a better error message.  This is not a copy/paste of `selectors._fileobj_to_fd`.

Reply:
Got it, sorry.